src/cascadia/WinRTUtils/ScopedResourceLoader.cpp

```diff
@@ -46,3 +46,14 @@ bool ScopedResourceLoader::HasResourceWithName(const std::wstring_view resourceN
 {
     return _resourceMap.HasKey(resourceName);
 }
+
+ScopedResourceLoader::ScopedResourceLoader(winrt::Windows::ApplicationModel::Resources::Core::ResourceMap map, winrt::Windows::ApplicationModel::Resources::Core::ResourceContext context) :
+    _resourceMap{ std::move(map) }, _resourceContext{ std::move(context) } {}
+
+ScopedResourceLoader ScopedResourceLoader::WithQualifier(const std::wstring_view qualifierName, const std::wstring_view qualifierValue) const

```

Comment:
Should we use zwstring_view here to prevent any unintended string-slices from being passed accidentally?

Reply:
done